src/main/java/uk/gov/mca/beacons/service/configuration/JacksonConfiguration.java

```diff
@@ -20,7 +21,7 @@ public ObjectMapper objectMapper() {
       true
     );
     objectMapper.registerModule(new JavaTimeModule());
-
+    objectMapper.setDateFormat(new StdDateFormat().withColonInTimeZone(true));
```

Comment:
@Ninamma @tm1303 just a note on this, you might be able to autowire in the Spring configured `ObjectMapper` and then configure that instance rather than creating a new `ObjectMapper` and configuring what is already set.

So something like:

```java
  @Bean
  public ObjectMapper objectMapper(ObjectMapper objectMapper) {
// Do beacons specific configuration

```

Reply:
will revisit this issue when we refactor the manual serializer